src/Qir/Runtime/public/QirRuntime.hpp

```diff
@@ -61,12 +61,6 @@ extern "C"
     // Utils
     // ------------------------------------------------------------------------
 
-    // Allocate a block of memory on the heap.
-    QIR_SHARED_API char* __quantum__rt__heap_alloc(uint64_t size); // NOLINT
```

Comment:
This change doesn't seem to be in line with the rest of the PR.

Reply:
Yeah, it's not, but it's an annoying little thing that I missed more than a year ago and keep forgetting to fix. I felt it fit under the theme of "clean-up" but agree that it is not Rust related where every other change is. Should I separate it out to another PR?